src/VolleyManagement.Backend/VolleyManagement.Services/TournamentService.cs

```diff
@@ -413,6 +414,8 @@ public byte CalculateNumberOfRounds(TournamentSchemeEnum  scheme, int teamCount)
                 case TournamentSchemeEnum.PlayOff:
                     numberOfRounds = GetNumberOfRoundsByPlayOffScheme(teamCount);
                     break;
+                default: numberOfRounds = 0;
```

Comment:
Why to assign same value?

Reply:
because SonarLint needed a default, and in this all case, I did not see a main. Or I need make a default one of them?

Reply:
or can I make "if else" instead switch?

Reply:
Throw invalid op exception

Reply:
throw new InvalidOperationException("This Scheme does`t exist");? or better without message?

Reply:
ok